client/src/app.jsx

```diff
@@ -1,17 +1,88 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import utils from '../utils/utils.js';
+import urls from '../../config.js';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      url: window.location.href,
+      ticker: 'ABCD',
+      chartUrl: null,
+    };
+  }
+
+  componentDidMount() {
+
+    // create a script tag with the appropriate URL, and add it to the DOM
+    let script = document.createElement('script');
+    script.src = "http://localhost:4444/bundle.js";
+    document.querySelector('body').appendChild(script);
+
+
+    // TODO: how to go about reading the ticker from the url and sending it to the microservice?
```

Comment:
Maybe you can use window.location.pathname to find the ticker, but default to ABCD?

Reply:
neat! that should work ...